lua/lualine/init.lua

```diff
@@ -239,33 +132,34 @@ local function tabline() return statusline(config.tabline, true) end
 
 local function setup_theme()
   local async_loader
-  async_loader = vim.loop.new_async(vim.schedule_wrap(function()
-    local function get_theme_from_config()
-      local theme_name = config.options.theme
-      if type(theme_name) == 'string' then
-        local ok, theme = pcall(require, 'lualine.themes.' .. theme_name)
-        if ok then return theme end
-      elseif type(theme_name) == 'table' then
-        -- use the provided theme as-is
-        return config.options.theme
-      end
-      vim.api.nvim_echo({
-        {
-          'theme ' .. tostring(theme_name) .. ' not found, defaulting to gruvbox',
-          'ErrorMsg'
-        }
-      }, true, {})
-      return require 'lualine.themes.gruvbox'
-    end
-    local theme = get_theme_from_config()
-    highlight.create_highlight_groups(theme)
-    vim.api.nvim_exec([[
+  async_loader = vim.loop.new_async(vim.schedule_wrap(
+                                        function()
+        local function get_theme_from_config()
+          local theme_name = config.options.theme
+          if type(theme_name) == 'string' then
+            local ok, theme = pcall(require, 'lualine.themes.' .. theme_name)
+            if ok then return theme end
+          elseif type(theme_name) == 'table' then
+            -- use the provided theme as-is
+            return config.options.theme
+          end
+          vim.api.nvim_echo({
+            {
+              'theme ' .. tostring(theme_name) ..
+                  ' not found, defaulting to gruvbox', 'ErrorMsg'
+            }
+          }, true, {})
+          return require 'lualine.themes.gruvbox'
+        end
+        local theme = get_theme_from_config()
+        highlight.create_highlight_groups(theme)
+        vim.api.nvim_exec([[
     augroup lualine
     autocmd ColorScheme * lua require'lualine.utils.utils'.reload_highlights()
     augroup END
     ]], false)
-    async_loader:close()
-  end))
+        async_loader:close()
```

Comment:
The indents for commands look wrong

Reply:
my lua-format broke or something? I will apply formatting today